__init__.py

```diff
@@ -349,7 +349,7 @@ def _insert(self, name=None, value=None):
 
     def _remove(self, name=None, confirmed=None):
         if not name:
-            self.q.put([Action.ask_input, _("Remove which password?"), "", "remove"])
+            self.q.put([Action.ask_input, _("Which one of the passwords should be removed?"), "", "remove"])
```

Comment:
"Which one of the passwords should be removed?" is in a way different style than earlier strings like "Which password do you want to copy?". Maybe just "Which password should be removed?" Also, this is singular.